### ISSUE TITLE
Keep manual added templates during sync job

### ISSUE THREAD

**Issue by chrkli193** [Icinga/icingaweb2-module-director#752]
At the moment I import all our hosts from a mysql database. Based on the os, location, ... I set some templates during this import job. But when I add a template later manual to a host this will be deleted after the next sync job. 
The manual added templates should also been kept during an import run.

**Comment by mbassett** [comment]
Make sure your Sync job is set to Merge not replace and that should be the behavior you get.

**Comment by chrkli193** [comment]
Yes it is set to merge. All manual added templates get deleted during sync.

**Comment by chrkli193** [comment]
Is that working in other setups? Maybe it's a local problem?

**Comment by eyesonlyone** [comment]
I have the same problem... I don't think this is expected behaviour.
